src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -521,4 +521,7 @@
   <data name="Autoselect_disabled_due_to_member_declaration" xml:space="preserve">
     <value>Autoselect disabled due to member declaration</value>
   </data>
+  <data name="suggested_name" xml:space="preserve">
```

Comment:
capitalization of the resource name usually matches the capitalization of the string.